app/controllers/admin/works_controller.rb

```diff
@@ -119,7 +121,42 @@ def download_ohms_xml
     send_data @work.oral_history_content!.ohms_xml_text,
       :type => 'text/xml; charset=UTF-8;',
       :disposition => ContentDisposition.format(disposition: "attachment", filename: "#{@work.oral_history_content!.ohms_xml.accession}.xml")
-      #:disposition => "attachment; filename=#{@work.oral_history_content!.ohms_xml.accession}.xml"
+  end
+
+  # PATCH/PUT /admin/works/ab2323ac/submit_ohms_xml
```

Comment:
We COULD try to combine these actions with the existing ones for OHMS XML file -- since it is kind fo similar, need to add/remove/download "a file", we could try to it into a generic set of actions for adding/removing/downloading files, that could be re-used. 

BUT I'm honestly not sure if that is over-engineering or not; I actually think this is probalby just fine, and we should just go ahead with it rather than spending more time on this, but wanted to mention it just to share knowledge/thoughts with "the team" (Ie you and me).  But I think probably no need and this is good!

Reply:
Fair enough. I'm usually the one who always insists on super-DRYing everything but in this case -- yeah, this is fine.